src/smart-components/common/catalog-breadcrumbs.tsx

```diff
@@ -5,10 +5,18 @@ import ConditionalLink from '../../presentational-components/shared/conditional-
 import { BreadcrumbFragment } from '../../redux/reducers/breadcrumbs-reducer';
 import { CatalogRootState } from '../../types/redux';
 
-const CatalogBreadcrumbs: React.ComponentType = () => {
-  const fragments = useSelector<CatalogRootState, BreadcrumbFragment[]>(
-    ({ breadcrumbsReducer: { fragments } }) => fragments
-  );
+export interface CatalogBreadcrumbsProps {
+  breadcrumbfragments?: BreadcrumbFragment[];
+}
+const CatalogBreadcrumbs: React.ComponentType<CatalogBreadcrumbsProps> = ({
+  breadcrumbfragments = []
+}) => {
+  const fragments = breadcrumbfragments?.length
+    ? breadcrumbfragments
+    : useSelector<CatalogRootState, BreadcrumbFragment[]>(
+        ({ breadcrumbsReducer: { fragments } }) => fragments
+      );
```

Comment:
This ternary doesn't stand out as much to me, but I still feel like stretching a ternary over multiple lines is confusing.

Reply:
The TS code is verbose... however, I have updated both places where ternary op was used in this PR.